libs/ledger-live-common/src/config/sharedConfig.ts

```diff
@@ -1,6 +1,13 @@
 import { ConfigSchema } from "@ledgerhq/live-config/LiveConfig";
-import cosmosConfig from "../families/cosmos/config";
+import { cosmosConfig } from "../families/cosmos/config";
+import { bitcoinConfig } from "../families/bitcoin/config";
+import { evmConfig } from "../families/evm/config";
 
 const liveCommonConfig: ConfigSchema = {};
 
-export default { ...liveCommonConfig, ...cosmosConfig } as ConfigSchema;
```

Comment:
it sounds like this should be a "generated" file. (it's good when we can have all the "multiplexer" entrypoints in generated/ because it's a pattern we will eventually want to opt out when we truly modularize live-common)

Reply:
Can be discussed further be overall we try move away from this generated file pattern

Reply:
It's true that it would be better to use the 'generated' file pattern. In this case, we should modify 'ledger-live-common/scripts/sync-families-dispatch.mjs'. You can sync with Gaetan regarding this improvement. However, I consider that to be an enhancement for a separate task. I am okay with this PR as it is.